src/drivers/cmakeDriver.ts

```diff
@@ -333,6 +333,25 @@ export abstract class CMakeDriver implements vscode.Disposable {
         }
     }
 
+    /**
+     * Get the environment variables that should be set at CPack and packaging time.
+     */
+    async getCPackCommandEnvironment(): Promise<Environment> {
```

Comment:
Just as a note, we should test that the `NOTE` log statement only shows up in reasonable places. I think it likely does from reading all of the other code, but just making sure.